### DISCUSSION TITLE
Ability to align EP's weather tile to the left when the taskbar is centered with start

### DISCUSSION THREAD

**Discussion by prakharb5** [valinet/ExplorerPatcher#1007] [Ideas]
On Windows 10, if I center the taskbar along with start button, EP's weather tile on the taskbar is still aligned to the right. Having the ability to align it to the left would be really great, since it would mimic the weather icon behaviour on Windows 11

**Comment by valinet** [comment]
Ofc, I was wondering how long until someone mentioned it. Yeah, would make a nice option, kind of difficult to do; it will probably make its way at some point.

**Reply by prakharb5** [reply]
Good to know, looking forward to it :D

**Reply by valinet** [reply]
The widget can still be used just fine until then, I mean, Microsoft came with this left-alignment thing a few weeks ago and suddenly it's the next big thing...

**Reply by prakharb5** [reply]
Well, yeah. But currently enabling it just put it on the right, and it looks *very* unbalanced to me.

The left-alignment is actually a good change, not gonna lie.
